src/lib/Repository/UiLocationService.php

```diff
@@ -123,7 +135,7 @@ public function trashLocationAndReturnParent(Location $location)
      */
     public function canRemoveLocation(Location $location)
     {
-        if ($location->contentId == 1) {
+        if ($this->isRootLocation($location)) {
```

Comment:
you could just write:
```php
return !$this->isRootLocation($location);
```

Reply:
but how will it do the permission check that comes afterwards?

Reply:
ah yeah missed this one, so this comment only applies to `canMoveLocation` :)